src/utils/div.ts

```diff
@@ -0,0 +1,11 @@
+export function toggleElement (event: React.ChangeEvent<{}>, value: string): void {
+  const val = (event.target as HTMLInputElement).value;
+  const element = document.getElementById(value);
+  if (element != null) {
+    if (val === "yes") {
+      element.style.display = "block";
+    } else {
+      element.style.display = "none";
+    }
+  }
+};
```

Comment:
I get what you're doing here, but it is circumventing the React framework.
Why can't we accomplish this by checking the value in state and conditionally rendering the element?
```suggestion
```

Reply:
resolved in 3a33c37